esmvaltool/diag_scripts/shared/statistics.ncl

```diff
@@ -197,6 +198,11 @@ begin
   date := cd_calendar(field&time, 0)
   year := date(:, 0)
   month := date(:, 1)
+  if (isatt(field&time, "calendar")) then
+    cal = field&time@calendar
+  else
+    cal = "standard"  ; this is the default setting for 'days_in_month'
+  end if
```

Comment:
```suggestion
  cal = field&time@calendar
```

I think this `if` loop can be removed since there's a check for the calendar above:

https://github.com/ESMValGroup/ESMValTool/blob/0f3441dd495de6b3d5ae75785deed58520a59840/esmvaltool/diag_scripts/shared/statistics.ncl#L181-L185